### PR TITLE
Changes in entities

### PR DESCRIPTION
**Pull request by michalpavuk**
See 2d2000b commit message for more info.

### CODE REVIEW

File: src/main/java/cz/muni/fi/pa165/heroes/entity/Actor.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        if (hitpoints <= 0) throw new IllegalArgumentException("An actor must have more than 0 hitpoints");
+        this.hitpoints = hitpoints;
+    }
+
+    public int getDamage() {
+        return damage;
+    }
+
+    public void setDamage(int damage) {
+        this.damage = damage;
```
[the remark this answers is not in the document]
Comment (michalpavuk) [reply src/main/java/cz/muni/fi/pa165/heroes/entity/Actor.java]: Yes. If he is a healer for example.

File: src/main/java/cz/muni/fi/pa165/heroes/entity/Skill.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    public List<Affinity> getEffects() {
+        return effects;
+    }
+
+    public int getBaseDamage() {
+        return baseDamage;
+    }
+
+    public void setBaseDamage(int baseDamage) {
+        this.baseDamage = baseDamage;
```
[the remark this answers is not in the document]
Comment (michalpavuk) [reply src/main/java/cz/muni/fi/pa165/heroes/entity/Skill.java]: Healing skill

File: src/main/java/cz/muni/fi/pa165/heroes/entity/Skill.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        return name;
+    }
+
+    public void setName(String name) {
+        if (name == null) throw new IllegalArgumentException("Name of a skill can't be null");
+        this.name = name;
+    }
+
+    public List<Affinity> getEffects() {
+        return effects;
```
[the remark this answers is not in the document]
Comment (michalpavuk) [reply src/main/java/cz/muni/fi/pa165/heroes/entity/Skill.java]: Will do.

File: src/main/java/cz/muni/fi/pa165/heroes/entity/Affinity.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+ * recommended to join the quest regarding this monster.
+ */
+@Entity
+public class Affinity {
+
+    @Id
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
```
Comment (jstrmen) [inline src/main/java/cz/muni/fi/pa165/heroes/entity/Affinity.java]: Maybe name could be Enum.

Comment (jstrmen) [inline src/main/java/cz/muni/fi/pa165/heroes/entity/Affinity.java]: Maybe name could be Enum.

**Pull request merged**